client/navigation/style.scss

```diff
@@ -7,11 +7,17 @@
 
 	#wpadminbar,
 	#adminmenuwrap,
+	.woo-nav-tab-wrapper,
```

Comment:
This is a little bit risky.  I've noticed some of the other extensions using the woo nav tab wrapper while working on https://github.com/woocommerce/woocommerce-admin/issues/5335.

This is also used on several other core pages, like the marketplace.

<img width="477" alt="Screen Shot 2020-11-17 at 4 07 51 PM" src="https://user-images.githubusercontent.com/10561050/99450709-1b955b80-28ef-11eb-947b-13ace49872c9.png">

I think this is going to require more thought before we hide all of these nav tab wrappers since they can easily be reused or have items added/removed without the navigation being aware.

Reply:
You're right @joshuatf , that selector was way too broad 😱.

I think I figured out a way to narrowly target just the tabs within Settings via a body class, and I've pushed that change. It actually does _not_ apply the styles to any React-based pages within Settings (ie Analytics), but I'm noticing they didn't have the primary tabs to begin with.

> I've noticed some of the other extensions using the woo nav tab wrapper while working on #5335.

Let me know if you still think this change could still have issues with other extensions as mentioned^